sympy/core/tests/test_expand.py

```diff
@@ -230,6 +230,9 @@ def test_expand_arit():
     W = W.expand()
     assert W.has(-1672280820*x**15)
 
+def test_expand_mul():
+    e = Mul(2, 3, 1/x, evaluate = False)
+    assert e.expand() == 6/x
```

Comment:
There should also be tests for a pure `Mul(2, 3, evaluate=False)` and maybe `Mul(2, Rational(1, 3), evaluate=False)`.

Reply:
Sure thing. I've added them both.

Reply:
What is the policy on resolving conversations? Should I resolve this one?

Reply:
You can if you like. It's mainly useful if there are lots of them.